pkgs/development/libraries/libarchive/default.nix

```diff
@@ -44,6 +44,12 @@ stdenv.mkDerivation (finalAttrs: {
       url = "https://github.com/libarchive/libarchive/commit/3bd918d92f8c34ba12de9c6604d96f9e262a59fc.patch";
       hash = "sha256-RM3xFM6S2DkM5DJ0kAba8eLzEXuY5/7AaU06maHJ6rM=";
     })
+    (fetchpatch {
+      name = "revert-suspicious-commit-from-known-bad-actor.patch";
+      url = "https://github.com/libarchive/libarchive/commit/e37efc16c8665f405182dd77eb1bbb32376f5484.patch";
+      hash = "sha256-sjj0J7jkKr7+sRdww3awDOHsL8vlNs/cG5FVLSPIEWc=";
+      revert = true;
```

Comment:
There is a following PR to make it use `safe_printf` again, we could use that patch directly, instead of hand-crafting the revert patch. https://github.com/libarchive/libarchive/pull/2101

Reply:
This my initial thought as well but there is additional concerns on those changes see #300114

Reply:
I have re-opened the first PR now that things are a little clearer than they were. I'm closing this one.